python/tvm/relay/frontend/tensorflow2.py

```diff
@@ -38,12 +38,20 @@
 from .common import infer_type as _infer_type
 
 from .tensorflow_ops import _convert_map as _convert_map_common
-from .tensorflow_ops import _need_prelude_for_shape_inference
+from .tensorflow_ops import _get_more_static_shape_rank
+from .tensorflow2_ops import _convert_map as _convert_map_tf2
+from .tensorflow2_ops import _need_prelude_for_shape_inference
 
 from ..ty import Any
 
 __all__ = ["from_tensorflow"]
 
+# A map to record tensor list write ops and input tl/tensor indices
+# Value is (index of tensor list, index of written node)
+_tensor_list_write_ops = {
+    "TensorListSetItem": (0, 2),
```

Comment:
is there any reason why you chose 0 and 2 here?

Reply:
This is the input tensorlist and tensor indices defined in tensorflow API